SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.h

```diff
@@ -97,6 +97,19 @@ class SOFA_HELPER_API Removed : public ComponentChange
     }
 };
 
+class SOFA_HELPER_API Moved : public ComponentChange
+{
+public:
+    Moved(std::string  sinceVersion, std::string fromPlugin, std::string toPlugin) {
+        std::stringstream output;
+        output << "This component has moved from " << fromPlugin << " to " << toPlugin << " since SOFA " << sinceVersion << "."
```

Comment:
```suggestion
        output << "This component has been moved from " << fromPlugin << " to " << toPlugin << " since SOFA " << sinceVersion << "."
```